src/components/ProcessDocumentScreen/ProcessDocumentScreen.tsx

```diff
@@ -122,96 +101,56 @@ export const ProcessDocumentScreen: FunctionComponent<ProcessDocumentScreen> = (
                 </div>
               )}
             {failPublishedDocuments && failPublishedDocuments.length > 0 && queueState !== QueueState.ERROR && (
-              <div className="container">
-                <div className="text-cloud-500 font-medium text-lg my-6">
-                  {failPublishedDocuments.length}
-                  {isIssuingFlow ? " Document(s)" : " Document"} Failed
-                </div>
-                <div className="bg-rose-400 p-3 flex flex-col">
-                  <div className="flex">
-                    <XCircle className="text-rose" />
-                    <div className="text-rose ml-2 flex-grow" data-testid="error-message">
-                      These documents failed to publish due to some errors. Kindly rectify and try publishing again.
-                    </div>
-                    <Button className="bg-white text-rose hover:bg-cloud-100">
-                      <a
-                        download={generateFileName({
-                          network: config?.network,
-                          fileName: "error-log",
-                          extension: "txt",
-                          hasTimestamp: true,
-                        })}
-                        href={`data:text/plain;charset=UTF-8,${JSON.stringify(formattedErrorLog, null, 2)}`}
-                      >
-                        <div className="flex">
-                          <Download />
-                          <div className="text-rose ml-2">Download Error Log</div>
-                        </div>
-                      </a>
-                    </Button>
-                  </div>
+              <>
+                <ErrorCard
+                  title={`The document(s) could not be ${isIssuingFlow ? "issued" : "revoked"} at this time.`}
+                  description={
+                    "Please contact TradeTrust via email or client representative to resolve your issue. Alternatively, please try again."
+                  }
+                  buttonLink={
+                    <a className="text-cerulean" onClick={() => processDocuments(type)}>
+                      Try Again
+                    </a>
```

Comment:
i guess we could pass in `processDocuments` instead? so no need to think about what markup to pass in as much as possible